selfdrive/thermald/thermald.py

```diff
@@ -134,17 +134,16 @@ def handle_fan_uno(controller, max_cpu_temp, fan_speed, ignition):
 def handle_fan_tici(controller, max_cpu_temp, fan_speed, ignition):
   global last_ignition
 
-  controller.neg_limit = -(80 if ignition else 30)
-  controller.pos_limit = -(30 if ignition else 0)
-
   if ignition != last_ignition:
     controller.reset()
 
-  fan_pwr_out = -int(controller.update(
-                     setpoint=75,
-                     measurement=max_cpu_temp,
-                     feedforward=interp(max_cpu_temp, [60.0, 100.0], [0, -80])
```

Comment:
This line is the problem. 
Specifically "max_cpu_temp" should not be in the feedforward path. This is feedback. 

bonus improvement is that using the target here is correct. however the target is hardcoded so previously I just shifted everything by 30 corresponding to the setpoint.